### PR TITLE
init_project

### PR DESCRIPTION
**Pull request by chan227**

### CODE REVIEW

**Comment by chan227** [comment]
ready

**Comment by chan227** [comment]
Ready

**Comment by chan227** [comment]
ready

**Comment by chan227** [comment]
ready

**Comment by chan227** [comment]
ready

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class ApplicationController < ActionController::Base
+  before_action :switch_locale
+
+  def hello
```
Comment (manhnt-0960) [inline app/controllers/application_controller.rb]: let's remove this method because never used

**Comment by chan227** [comment]
ready

**Comment by chan227** [comment]
ready

**Pull request merged**